sql/moz-fx-data-shared-prod/fenix_derived/feature_usage_events_v1/query.sql

```diff
@@ -938,7 +947,27 @@ SELECT
       WHEN home_page_customize_home_clicked > 0
         THEN client_id
     END
-  ) AS home_page_customize_home_clicked_users
+  ) AS home_page_customize_home_clicked_users,
+  distribution_id,
```

Comment:
this `distribution_id` is a bit hidden right now among all the aggregations. Would it make sense to make it live just under the `is_default_browser`? We could keep all the grouping keys together this way and the query easier to read.